### DISCUSSION TITLE
Quick note of thanks!

### DISCUSSION THREAD

**Discussion by tannenbe** [jessedp/tablo-tools-electron#207] [General]
Just a **quick note of thanks** for all your work on tablo tools. I installed the app on my Win 11 Pro laptop and was able to successfully download (and view) a test episode of the 52nd season of The Price Is Right, hosted by Drew Carey at their new studio in Glendale CA. 

The whole process was fast and straightforward.. w/o reading much of anything but finding my way around your cool app. 

Whoo hoo! 

My plan is to record Oct 18 2023 episode then download using your tools.. I attended a taping back in Aud and it airs on Oct 18.

I'll use slow motion to capture images of me 2 rows back from Contestants Row.. I didn't get called to "Come on down" but I plan to make icons / memes out of any usable images of me :-)

And yes.. I did see you were a Whose Line Is It Anyway fan ... hence my notice of seeing Drew in person..

**Comment by jessedp** [comment]
I appreciate the kind words! Not only am I a _Whose Line_ fan, I love game shows and watch _Let's Make a Deal_ and _The Price is Right_  religiously, so will certainly be watching that day. Cheers!

**Reply by tannenbe** [reply]
Wow!! That's fantastic! I hope you don't mind my post about my experience.

BTW, Wayne Brady is an amazing entertainer too! They also tape LMAD in the same new studio, though from Drew's jokes during our taping, LMAD wrapped their taping up and Haven Studios is all TPIR... 

I've been watching TPIR since it (re)started in 1972,,, I also saw a taping with Bob Barker in 2001 in Television City and in BB studio.. this was before Google Maps and my driving skills got me lost and I barely made it on time so I was way in the back and that studio was big.

This new facility is all high tech, The new studio audience holds only ~ 200 people so more chance of getting called.

RIght now, Freemantle Productions borrows space from a next door Asian Church and we park and process Gameshow paper work and do a little group interviews for the producers to pick the contestants. We all get our name tags for TV (and Drew) and importantly, our contestant number.

The whole process takes 5 hrs.. from the time you park to the time you leave. We wait around a bit but you're in a pepped up crowd and everyone is excited to be there so time goes pretty fast. We also have our cell phones to keep busy while waiting to go into the studio,

I used mine to take pix.. which are attached. They also tell you to bring snacks or sodas because there is no food or vending there BUT NO food or drink is allowed inside the studio.. which is very understandable!!!

Contestant waiting area where we signed paper work, waited for row by row interviews in another area by the producers and then waited till we walked across the parking lot to the adjacent studio
![20230824_132204](https://github.com/jessedp/tablo-tools-electron/assets/52576929/6f1bb113-1ee0-4ab3-ae2a-c0a1a81b1012)

Walking across the parking lot to the adjacent studio
![20230824_142400](https://github.com/jessedp/tablo-tools-electron/assets/52576929/e7f6009c-1e7d-4e61-8439-190fcfb1bc03)
![20230824_142558](https://github.com/jessedp/tablo-tools-electron/assets/52576929/2fc8e435-4bdc-41d6-b9a2-67d6e00875d8)

And then the MANDATORY cell phone drop off :-)
![20230824_142633](https://github.com/jessedp/tablo-tools-electron/assets/52576929/1511fc6f-cef7-48ed-9176-c735c0307256)

I took this while in the Church before we went in.. 
![20230824_122606](https://github.com/jessedp/tablo-tools-electron/assets/52576929/b592eaf4-1e15-4338-8441-13d0b60e357d)

Then once in the studio, they planned out EVERYONE's seat location, using our contestant #,

Then, just before taping began, George Grey comes out to chat with us and then he tells us to remove our contestant # sticker so just our TPIR price tag name appears.

The taping was exciting and fascinating.. you are very close to everything.. 

If you ever find yourself in the Los Angeles area, plan ahead and attend a taping if they're doing so.
